Modules/socketmodule.c

```diff
@@ -876,7 +870,22 @@ init_sockobject(PySocketSockObject *s,
 {
     s->sock_fd = fd;
     s->sock_family = family;
+
+#ifdef __linux__
```

Comment:
See the bpo. The bug is not specific to Linux, FreeBSD is also affected. Maybe also other OSes?

Reply:
Fixed.